guides-markdown/multisig-wallet.md

```diff
@@ -0,0 +1,359 @@
+# Using bcoin wallet node for multisig wallet
+
+```post-author
+Joe Miyamoto <joemphilips@gmail.com>
+```
+
+```post-description
+How to use bcoin with a multisig wallet and account. For both as a hot-wallet and as a watch-only-wallet.
+```
+
+## How it works
+
+Bitcoin multisig account is an account which holds more than 1 public key,
+and generates P2SH or P2WSH address for those public key.
+When you want to use those funds,
+we must create multisig transaction which has been signed by private key corresponds to those public key,
+see [our multisig transaction guide](http://bcoin.io/guides/multisig-tx.html) for more detail about multisig transaction itself.
+
+In this guide, we take a step little further and see a whole lifecycle of the multisig account in a more practical way.
+
+## Usecases
+
+### 1. Personal wallet which is secure against key compromisation
+
+Usually in a cryptocurrency world, when you lose your master key and it's seed for the wallet there are no way to recover your funds.
+But by splitting your secret keys into ...
+
+1. your own server
+2. local hardware you own
+3. your personal vault with paper wallet.
+
+you can backup your funds unless more than two keys were compromised at the same time.
+In this case, you may use 1. and 2. for usual spending.
+You only need 3 when you lost your secret key for 1 and 2
+
+### 2. Wallet held by more than one person
+
+Some kind of funds are desirable to be held by several people.
+For example, funds for a corporation or for a crowdfunding project.
+We can mitigate the risk of fund loss by utilizing a multisig in the case such as ...
+
+1. Person who is in charge has changed.
+2. Or disappeared suddenly.
+3. Or lost his secret-key.
+4. Or be stolen the secret-key
+5. Or tried to thief all funds for his personal use.
+
+## Walk through
+
+ This tutorial includes following step.
+ 1. setting up a bcoin wallet server with https.
+ 2. setting up a hot multisig wallet for personal use.
+ 3. setting up a watch-only multisig wallet.
+ 4. receive incoming information from the wallet server.
+ 5. spend funds from the wallet.
+ 6. recovering the wallet
+
+### 1. Setting up bcoin wallet server with https.
```

Comment:
Well, this is because I was thinking that this watch-only wallet works as kind of a "semi-trusted watch-tower service" for every co-signer.
I mean, the scenario I was assuming is that co-signer don't have to run a wallet server in the local, instead they register their xpub onto this server and listen to it directly.
I made it this way because this will likely make things more simple when e.g. some co-signer made a intent to pay from the wallet. wallet server can directly push notification to every co-signer. (Possibly through plugin.)
I will certainly clarify this logic in the document. And mention that https is not mandatory as you pointed out.